app/models/Item.rb

```diff
@@ -34,4 +34,13 @@ def average_fulfillment_time
       return nil
     end
   end
+
+  def ordered?
+    OrderItem.find_by(item: self)
+  end
+
```

Comment:
So if `ordered?` doesn't find anything - it returns nil? Because in my experience ActiveRecord returns an empty array when it finds nothing.

Which would evaluate as truthy